code/modules/surgery/organs/organ_internal.dm

```diff
@@ -174,13 +183,18 @@
 	Restart()
 	..()
 
+/obj/item/organ/internal/heart/necrotize()
```

Comment:
For now, this is the only thing we'd like included in this PR, from a balance altering standpoint.